docs/data-access/icgc-25k-data.md

```diff
@@ -0,0 +1,100 @@
+
+The aim was to analyze multiple cancer types and share open access data such as simple somatic mutations, copy number alterations, structural rearrangements, gene expression, microRNAs, DNA methylation and exon junctions.
+
+### Relocated ICGC 25K Data
+
+Although the interactive data portal was shut down, data from the project remains available. The final release data, as well as the PCAWG project data are now hosted by ICGC ARGO. This data is [available to authorized users via an SFTP](#accessing-icgc-25k-release-data).
+
+Files from other contributing projects are all hosted by ICGC's partner repositories. To access this data, you will need to identify which repository hosts the data you are looking for and then request the data through their service. A [mapping file is available](#mapping-icgc-legacy-data-to-external-repositories) to download below which maps ICGC 25K file IDs to their current hosted location.
+
+## Accessing ICGC 25K Release Data
+
+An SFTP server is available to access ICGC Release Data and PCAWG data.
+
+The server hosts three data directories with the following data:
+
+- `/release_28` - This is the Data Portal data Release 28 of the International Cancer Genome Consortium (ICGC).
+- `/PCAWG` - Analysis results from the PCAWG study.
+- `/Supplemental` - Corrected clinical metadata and RNA-Seq raw read counts for projects LICA-FR and PRAD-UK.
+
+Only users with DACO approval are able to access the SFTP server. If you previously had DACO access for ICGC 25K data you will continue to have permission to access the SFTP server. If you require DACO approval please see the documentation on [applying for DACO access](./daco/applying.md).
+
+### SFTP Connection Details
+
+The SFTP server is located at:
+
+- **Host**: `legacy-icgc-sftp.platform.icgc-argo.org`
+- **Port**: `2222`
+
+Authentication to the server is done using username and password:
+
+- **Username**: The email address that was approved for DACO access. This is the account you would use to log into the [ARGO platform](https://platform.icgc-argo.com).
+- **Password**: ICGC API Key. This is available on your ARGO Platform [profile page](https://platform.icgc-argo.com/user).
+
+You can connect to this server using any SFTP client of your choice. For a free client you consider [FileZilla](https://filezilla-project.org/download.php?type=client).
+
+## Partner Repositories with ICGC 25K File Data
+
+ICGC file data is hosted across the following repositories.
+
+If you know the specific files from ICGC that you want to access, consult the provided [Mapping File](#mapping-icgc-legacy-data-to-external-repositories) to discover which repositories host that file.
+
+### EGA
+
+Raw sequences submitted to ICGC or reprocessed apart of PCAWG are hosted across multiple EGA datasets managed by the data access committee [EGAC00001000010](https://ega-archive.org/dacs/EGAC00001000010).
```

Comment:
is this a different DACO? I thought it was all the same? If all the same indicate it is the same, otherwise I'm thinking shoot I need to do a new application.

Maybe "managed by the ICGC 25k data access committee."

Reply:
`Raw sequences submitted to ICGC or reprocessed apart of PCAWG are hosted across multiple EGA datasets managed by the data access committee [EGAC00001000010], which is included with DACO approval`?

Reply:
`ICGC 25k DACO approval includes access to raw sequences submitted to ICGC and reprocessed PCAWG files hosted on EGA. Data is spread across multiple datasets under data access committee [EGAC00001000010]`?

Reply:
ICGC 25k DACO approval includes access to raw sequences submitted to ICGC and reprocessed PCAWG files hosted on EGA. Available datasets are listed under the data access committee [EGAC00001000010](https://ega-archive.org/dacs/EGAC00001000010).

Reply:
Updated:
https://github.com/icgc-argo/argo-docs/blob/72d71575e18f3efc54a8dfd8dce03b32465ccf6f/docs/data-access/icgc-25k-data.md?plain=1#L57